chainercv/visualizations/vis_instance_segmentation.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import division
+
+import numpy as np
+
+from chainercv.visualizations.vis_semantic_segmentation import _default_cmap
+
+
+def vis_instance_segmentation(
```

Comment:
I want this to be consistent with `vis_bbox`.
This means it should take `img` together with annotations.

Reply:
i got comfused with vis_semantic_segmentation. why doesn't it have vis_image in vis_semantic _segmentation?

Reply:
i got comfused with vis_semantic_segmentation. why doesn't it have vis_image in vis_semantic _segmentation?

Reply:
I updated.

Reply:
In semantic segmentation, an annotation is displayed in a different plot from an image.
In the case of detection and instance segmentation, annotations are displayed on top of an image.
That is why taking image is mandatory for `vis_instance_segmentation`.